### PR TITLE
puts cloak in suit slot

### LINKED ISSUE #13309: Cloaks can't actually be worn in the exosuit slot

### PR DESCRIPTION
**Pull request by swankcookie**
fixes #13309 
:cl: swankcookie
tweak: Switches cloak usage to overclothing slot
/:cl:

### CODE REVIEW

**Comment by Incoming5643** [comment]
the fact that this fixes a recent changelogged change makes me think it probably doesn't need it's own log.

**Comment by swankcookie** [comment]
i just kinda had a "better safe than sorry" mindset. I agree though.

**Comment by Shadowlight213** [comment]
@swankcookie wew merge conflicts.

File: code/modules/clothing/suits/cloaks.dm [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	icon = 'icons/obj/clothing/cloaks.dmi'
 	icon_state = "qmcloak"
 	w_class = 2
-	slot_flags = SLOT_BACK
+	slot_flags = SLOT_OCLOTHING
```
Comment (Chronitonity) [inline code/modules/clothing/suits/cloaks.dm]: You shouldn't need to define the slot_flags at all if its a suit, it'll do it automatically unless it's overridden with something else as far as I know.

**Comment by swankcookie** [comment]
@Shadowlight213 what?

File: code/modules/clothing/suits/cloaks.dm (hunk 1 above)
Comment (swankcookie) [inline code/modules/clothing/suits/cloaks.dm]: fixed

**Comment by swankcookie** [comment]
@Shadowlight213 dey fixed

**Comment by Thunder12345** [comment]
I... Think something went wrong here

**Comment by swankcookie** [comment]
@Thunder12345 elaborate?

**Comment by Thunder12345** [comment]
slot_flags = SLOT_OCLOTHING
This line disappeared in the conflict fixing

**Comment by swankcookie** [comment]
nope. I got rid of it manually. Chronitonity said it wasn't neccesary. (the Suit parent has Oclothing already set.)

**Comment by swankcookie** [comment]
@Thunder12345 I just had too get rid of the slot_flags = SLOT_BACK

**Comment by Thunder12345** [comment]
Ahh, right

**Pull request merged**